### ISSUE TITLE
When loading session, are data samples already converted to microvolts? 

### ISSUE THREAD

**Issue by magdalenasabat** [open-ephys/open-ephys-matlab-tools#9]
as in the title

**Comment by medengineer** [comment]
No, you currently need to multiply each stream's data by its corresponding bitVolts value found in the 'structure' file of your recording. 

This should be done automatically, I will add it in the next update and close this issue when it's available.
